internal/processor/stream/test_approved_es_documents/testIntakeIntegrationSpanLinks.approved.json

```diff
@@ -97,6 +97,7 @@
             },
             "transaction": {
                 "id": "abcdef1478523690",
+                "representative_count": 1,
```

Comment:
Is "representative_count" equivalent of "sampling rate"? If so, why not call it the later?

Reply:
It's effectively the inverse sampling rate, except for old agents which send all transactions regardless of whether they're sampled: in this case we don't know the sample rate, so we set representative_count to 1 for every event we record.

Less important (since it could also be done with a runtime field): representative_count will be easier to deal with in https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-weight-avg-aggregation.html